src/modules/domain/healthcare/HealthcareModule.ts

```
@@ -0,0 +1,409 @@
+        },
+        identifier: match[1],
+      });
+    }
+
+    return medicalData;
+  }
+
+  private parseVitalSign(value: string, type: string): number {
+    switch (type) {
+      case 'blood_pressure':
+        const bpParts = value.split('/');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)